.github/workflows/commit_message.yml

```diff
@@ -0,0 +1,33 @@
+on: 
+  workflow_call:
+    outputs:
+      message:
+        description: "Skip tag checker"
+        value: ${{ jobs.check_skip_tags.outputs.message }}
+
+permissions:
+   contents: read
+
+jobs:
+  check_skip_tags:
+    name: Check for skips
+    runs-on: ubuntu-latest
+    outputs:
+      message: ${{ steps.skip_check.outputs.message }}
+    steps:
+      - name: Checkout scipy
+        uses: actions/checkout@v3
+        # Gets the correct commit message for pull request
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Check for skips
+        id: skip_check
+        run: |
+          set -xe
+          COMMIT_MSG=$(git log --no-merges -1)
+          RUN="1"
+          if [[ "$COMMIT_MSG" == *"[lint only]"* || "$COMMIT_MSG" == *"[docs only]"* ]]; then
+              RUN="0"
+          fi
```

Comment:
If we don't want the linter to run on '[docs only]', we could set `RUN="L"` for '[lint only]' then add a new job to the lint workflow to get the commit message and run if the output is `"1"` or `"L"`.